test/winners.js

```diff
@@ -25,4 +25,58 @@ weights = [
   { publisher: '17%', weight: 0.17 }
 ]
 tap.same(_.sortBy(Synopsis.prototype.winners(10, weights)),
-         [ '05%', '10%', '17%', '17%', '17%', '20%', '20%', '30%', '30%', '30%' ])
+  [ '05%', '10%', '17%', '17%', '17%', '20%', '20%', '30%', '30%', '30%' ])
+
+tap.same([], Synopsis.prototype.winners(92,
+  [
+    { publisher: 'brave.com', pinPercentage: undefined, weight: 0 },
+    { publisher: 'clifton.io', pinPercentage: undefined, weight: 0 }
```

Comment:
can we remove the 2nd item here? so that it matches the test case we had originally

Reply:
(see https://github.com/bsclifton/bat-publisher/commit/d3364cf17aa0c31922c21b17f972b936c92e45ee)

Reply:
@bsclifton additional test added here: https://github.com/brave-intl/bat-publisher/pull/35/files#diff-62377b611eef58dc8064a39a7b75112dR30

Only difference from original test case is that we are returning `[]` as opposed to `undefined` (winners should be an array in all cases)

Reply:
yup! that is perfect